mne/io/pick.py

```diff
@@ -60,7 +60,11 @@ def channel_type(info, idx):
     elif kind == FIFF.FIFFV_SYST_CH:
         return 'syst'
     elif kind in [FIFF.FIFFV_SEEG_CH, 702]:  # 702 was used before v0.11
```

Comment:
Did we agree to remove the check for 702 in 0.12? Maybe we should add ``XXX`` here so we don't forget to remove it. Isn't that what it is used for? And yeah, I should have put it when I did this.
Also, gamma map failure is back.